config/config.go

```diff
@@ -14,10 +14,11 @@ type PluginConfig struct {
 }
 
 type NRClientConfig struct {
-	Endpoint   string
-	ApiKey     string
-	LicenseKey string
-	UseApiKey  bool
+	Endpoint    string
+	ApiKey      string
+	LicenseKey  string
+	UseApiKey   bool
+	LowDataMode bool
```

Comment:
The new `LowDataMode` determines how data is _formatted_, rather than how data is _sent_. I believe it doesn't belong in `NRClientConfig`, but in a new config struct, such as `DataFormatConfig`. Then, the `PluginConfig` would have one field of type `DataFormatConfig`. What do you think?